lib/src/device_code_exception.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// An exception raised when OAuth2 device fails.
+class DeviceException implements Exception {
```

Comment:
Documentation link:
```suggestion
/// An exception raised when [DeviceAuthorizationGrant] fails.
class DeviceException implements Exception {
```

Can we be more specific about from where and when this is thrown... In what cases...

Reply:
should be fixed in f25abe5